data/clients.json

```diff
@@ -568,6 +568,7 @@
 		{
 			"name": "eggsampler/acme",
```

Comment:
Lego and Traefik (by way of Lego) are also V2 compatible and could use an update in `data/clients.json`.

Reply:
I'll update them, thanks!